app/models/reports/repository.rb

```diff
@@ -74,6 +74,7 @@ def initialize(regions, periods:)
     memoize def adjusted_patient_counts
       cumulative_assigned_patients_count.each_with_object({}) do |(entry, result), results|
         values = periods.each_with_object(Hash.new(0)) { |period, region_result|
+          next unless result.key?(period.adjusted_period)
```

Comment:
We don't want to attempt the subtraction here if there aren't any patient counts.